config/peerpods/mc-40-kata-remote-config.yaml

```diff
@@ -3,9 +3,9 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    app: peerpodconfig-openshift
+    app: example-kataconfig
     machineconfiguration.openshift.io/role: kata-oc
-  name: 40-worker-kata-remote-config
+  name: mc-40-kata-remote-config.yaml
```

Comment:
This diverges from the usual MC naming convention, can you explain the purpose of this move?  I suspect that given that MC names typically start with numbers, this would ensure the kata-remote-config MC is always applied last.

Reply:
My bad. The conversion using butane tool uses a config and I used an old config it seems. I'll commit the butane config as well